org.opent2t.sample.lamp.superpopular/com.insteon.lightbulb/js/thingTranslator.js

```diff
@@ -0,0 +1,137 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
```

Comment:
I don't think we need it. But all other hubs has it comment out and this goes back to Arjun's comments on we should probably keep everything consistent now and remove it later all together.